lektor/packages.py

```diff
@@ -257,7 +259,7 @@ def update_cache(package_root, remote_packages, local_package_path,
             shutil.rmtree(package_root)
         except OSError:
             pass
-        to_install = all_packages.items()
+        to_install = list(all_packages.items())
```

Comment:
I think we must use `iteritems()` for PY2 and `items() for PY3. if use `list()`, it cause slow at PY3